Microsoft.Toolkit.Services/Services/MicrosoftTranslator/DetectLanguageResponse.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Toolkit.Services.MicrosoftTranslator
+{
+    /// <summary>
+    /// Strong type for Detect Language Response
+    /// </summary>
+    /// <seealso cref="ITranslatorService.DetectLanguageAsync(string)"/>
+    public class DetectLanguageResponse : DetectLanguage
```

Comment:
Why three different classes? This seems like it can be combined in one class `DetectedLanguage`

Reply:
`DetectLanguageResponse` inherits from `DetectLanguage` Because the response of Detect Language response has the following properties: `Language`, `Score`, `IsTranslationSupported`, `IsTransliterationSupported `and `Alternatives`. The latter is a collection of object that contains the same exact properties except `Alternatives`, so I decide to create a base class to handle this situation.
`DetectLanguage` in turn inherits from `BaseDetectLanguage` (that probably should be name *DetectLanguageBase*) because the Translate response too contains an object that exposes these properties.
In short, I have created these three classes to avoid code duplication.

Reply:
Sounds reasonable. How do you feel about renaming these classes to `DetectedLanguage*` instead of `DetectLanguage*`?

Reply:
This makes sense. Done.